### PR TITLE
add and update several packages

### PR DESCRIPTION
**Pull request by dai-shi**
This adds 5 packages and renames one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +138,7 @@
 - [`react-speech-kit`](https://github.com/MikeyParton/react-speech-kit) Hooks for browser Speech Recognition and Speech Synthesis.
 - [`react-state-patterns`](https://github.com/mcclayton/react-state-patterns) Utility package for creating reusable implementations of React state provider patterns from hooks.
 - [`react-swipeable`](https://github.com/dogfessional/react-swipeable) React swipe event handler hook.
+- [`react-tracked`](https://github.com/dai-shi/react-tracked) Simple and fast global state with React Context. Eliminate unnecessary re-renders without hassle.
```
Comment (aarongarciah) [inline README.md]: `react-tracked` exposes any hook? Briefly looking at the docs doesn't seem like 🤔

Comment (dai-shi) [reply README.md]: It's like [constate](https://github.com/diegohaz/constate).
`createContainer` create hooks. Here's the api doc: https://react-tracked.js.org/docs/api

Comment (aarongarciah) [reply README.md]: Very cool :)

**Comment by aarongarciah** [comment]
Thanks!

**Pull request merged**